third_party/istio-1.4-latest

```diff
@@ -1 +1 @@
-istio-1.4.9
\ No newline at end of file
+istio-1.6.1
```

Comment:
Do we need to a create a directory istio-1.6-latest?

Reply:
oh OK I saw the directory was created. So do we still need this one?

Reply:
Yes, I think we should not remove it until we update `test-infra` to use istio-1.6. test-infra still specify istio-1.4 by arg like

https://github.com/knative/test-infra/blob/14ebc4a42974a95169fa9d65ccbb0b68661ae0ba/config/prod/prow/config_knative.yaml#L78
https://github.com/knative/test-infra/blob/14ebc4a42974a95169fa9d65ccbb0b68661ae0ba/config/prod/prow/jobs/config.yaml

The arg is used for `ISTIO_VERSION` in e2e-tests.sh
If we removed it, we cannot test istio-1.6 by `/test pull-knative-serving-istio-1.4-{no-}mesh` and testgrid.